Makefile

```diff
@@ -43,16 +46,34 @@ VERSION ?= main
 
 TAG_LATEST ?= false
 
+ifeq ($(TAG_LATEST), true)
+	IMAGE_TAGS ?= $(IMAGE):$(VERSION) $(IMAGE):latest
+else
+	IMAGE_TAGS ?= $(IMAGE):$(VERSION)
+endif
+
+ifeq ($(shell docker buildx inspect 2>/dev/null | awk '/Status/ { print $$2 }'), running)
+	BUILDX_ENABLED ?= true
+else
+	BUILDX_ENABLED ?= false
+endif
+
+CI ?= false
+
 # The version of restic binary to be downloaded for power architecture
 RESTIC_VERSION ?= 0.9.6
 
 CLI_PLATFORMS ?= linux-amd64 linux-arm linux-arm64 darwin-amd64 windows-amd64 linux-ppc64le
-CONTAINER_PLATFORMS ?= linux-amd64 linux-ppc64le linux-arm linux-arm64
-MANIFEST_PLATFORMS ?= amd64 ppc64le arm arm64
+BUILDX_PLATFORMS ?= linux/amd64,linux/arm64,linux/arm/v7,linux/ppc64le
+BUILDX_OUTPUT_TYPE ?= registry
```

Comment:
Can we make this default to `docker` and use `registry` only from [`hack/docker-push.sh`](https://github.com/vmware-tanzu/velero/blob/main/hack/docker-push.sh)?
Same with the `BUILDX_PLATFORMS`.

That would avoid creating the `local-all-containers` make target. WDYT?

Reply:
That sounds reasonable to me, I will refactor.